plugin-server/src/cdp/services/hogflow-executor.service.ts

```diff
@@ -94,6 +96,29 @@ export class HogFlowExecutorService {
             queue: 'hogflow',
         })
 
+        // If the flow is archived, check the stop_type
+        if (invocation.hogFlow.status === 'archived') {
```

Comment:
The nice thing about this function is that it is pretty "pure". So it just executes - it never thinks about statuses etc.

Personally I think we should do this at the consumer level which actually decides which things get executed https://github.com/PostHog/posthog/blob/master/plugin-server/src/cdp/consumers/cdp-cyclotron-worker-hogflow.consumer.ts#L23

We'll need to think about doing it there as the `HogFlowManager` is only loading flows from the DB that are `active` so this code will essentially never get hit hence modifying the getter logic and the consumers is going to be the right place for this